packages/edit-post/src/style.scss

```diff
@@ -61,9 +61,8 @@ body.block-editor-page {
 	@include wp-admin-reset( ".block-editor" );
 }
 
-.block-editor,
-// The modals are shown outside the .block-editor wrapper, they need these styles
-.components-modal__frame {
+// Target the editor UI but exclude the metaboxes and custom fields areas.
+.block-editor-styles-reset {
```

Comment:
I'm a bit concerned about all these reset classes added everywhere. Can't we replace the previous `.block-editor` with something more specific instead? 

Also, ideally we should try to remove that "reset" entirely by moving the styles to the specific components making the components more reusable without relying on global styles.

Reply:
`block-editor` contains the meta boxes and custom fields area. The only way would be to use a `:not()`, and I'm not sure if that would be any better. Another approach would be to target each UI section individually, as it worked before. Something like:
```
.editor-post-permalink,
.edit-post-sidebar,
.editor-post-publish-panel,
.editor-styles-wrapper,
.components-popover,
.components-modal__frame { ... }
```

but that would produce several lines of CSS selectors, while a single class would produce way less lines. No strong preferences on my side.

Reply:
> ideally we should try to remove that "reset" entirely by moving the styles to the specific components making the components more reusable without relying on global styles.

I'd agree. A bit out of the scope of this PR though.

Reply:
I have a preference for the `:not` approach if it works otherwise listing the selectors is better. That way we avoid introducing a `block-editor-styles-reset` class in a generic component like "modal". A Modal is agnostic of where it's being used.

Reply:
OK thanks, will try the `:not()` approach as soon as I have some time.